.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Environment variables should not be left empty. This can lead to confusion and potentially cause runtime errors if the variables are used without being properly defined. Ensure that values for 'GITHUB_TOKEN' and 'OPENAI_API_KEY' are correctly set in the environment to avoid issues.